README.md

```diff
@@ -1,91 +1,101 @@
 # enchannel-zmq-backend
 
-![enchannel version](https://img.shields.io/badge/enchannel-1.1-ff69b4.svg)
+[![GitHub release](https://img.shields.io/badge/version-latest-blue.svg)](https://github.com/nteract/enchannel-zmq-backend/releases)
```

Comment:
This "enchannel version" in reference to the "spec" in [enchannel](https://github.com/nteract/enchannel), which isn't so much a spec as "please make `Rx.Subject`s" for each of the Jupyter channels. That being said, I think we should switch to this format you have here.